tot/experiment.py

```diff
@@ -52,10 +52,11 @@ def __post_init__(self):
         if hasattr(self.data, "freq") and self.data.freq is not None:
             data_params["freq"] = self.data.freq
         self.params.update({"_data_params": data_params})
+        model_name = self.params.get("model", self.model_class).__name__
```

Comment:
`DartsForecastingModel` is a wrapper around any `darts` model, which is passed by a `model` param. For models that do not require `model` param the `model_name` defaults to the class name